app/Listeners/QueueMessage.php

```diff
@@ -29,20 +30,19 @@ public function __construct(Mailer $mail)
      */
     public function handle(QueueMessageRequest $event)
     {
-        $content = Message::prepareMessage($event->message, $event->competition);
+        $email = $event->email;
+        $type = $email->message->type;
```

Comment:
Do you need to include the `$key` as well.... so for like `reminder` types, there's like 5 of those...